src/services/organizeImports.ts

```diff
@@ -37,7 +37,14 @@ namespace ts.OrganizeImports {
             organizeImportsWorker(ambientModuleExportDecls, coalesceExports);
         }
 
-        return changeTracker.getChanges();
+        const calculatedChanges = changeTracker.getChanges();
+        return calculatedChanges.filter(fileTextChanges => {

```

Comment:
Why not do this in changeTracker.getChanges implementation itself?

Reply:
I considered that, but wasn't sure whether we wanted to apply it *everywhere*. I'm open to it if you think it's a good idea.

Reply:
i think its a good idea given: it will compact response and in turn requests too